packages/main/src/pages/protected/workspace/Workspace.jsx

```diff
@@ -22,6 +22,7 @@ import VideoChat from "../../../components/media-chat/VideoChat";
 import VoiceCall from "../../../components/media-chat/VoiceCall/VoiceCall";
 import { Sidebar, TopBar } from "../../../components/protected";
 import ExtraSidebar from "../../../components/protected/extra-sidebar";
+import WorkspaceLoading from "../../../components/general-loading/WorkspaceLoading";
 
 export default function Index() {
   // const { workspaceId } = useParams();
```

Comment:
You can remove this if it's not being used.

Reply:
Done